scripts/build.sh

```diff
@@ -527,6 +527,7 @@ echo "Checking past install"
 if [ ! -e "/storage/emulated/0/.ksu_completed_\$(getprop ro.build.date.utc)" ]; then
     echo "Installing KernelSU APK"
     pm install -i android -r /system/data-app/KernelSU.apk
+    appops set me.weishu.kernelsu ACCESS_RESTRICTED_SETTINGS allow
```

Comment:
no need it. Install with `-g` argument will grant all requested permissions

Reply:
Yes, of course I know.
I just tried to set the status of AppOps.

No need to do anything else.
https://github.com/tiann/KernelSU/blob/v0.5.7/manager/app/src/main/AndroidManifest.xml

I'll delete it later.

Reply:
@s1204IT Can you send the log ?